filesystem/config.pan.new

```diff
@@ -0,0 +1,804 @@
+# Template allow to configure partitionning with quite a lot of flexibility,
```

Comment:
I don't understand your question... There used to be a documentation about this template on the old wiki that may be worth migrating/rewriting... The basic idea is that you never have to replace `filesystem/config` that does a lot of things for you. You just need to define another template (called layout) where you define a few variables to steer `filesystem/config`.